users/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
-# Create your models here.
+# Mentor model
+
+class Mentor(models.Model):
```

Comment:
Should inherit from `AbstractUser`, learn more about authentication models [here](https://dontrepeatyourself.org/post/django-custom-user-model-extending-abstractuser/)